### PR TITLE
Changed screenshots as they the options have changed

### PR DESCRIPTION
**Pull request by msbemba**
Changed screenshots as they the options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8886

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/microsoft-365-solution-catalog.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ From here, you'll see cards for the following solutions:
     - [eDiscovery (Standard)](./get-started-core-ediscovery.md): Searches across content locations to identify, preserve, and export data in response to legal discovery requests and eDiscovery cases.
     - [eDiscovery (Premium)](overview-ediscovery-20.md): Builds on eDiscovery capabilities by providing intelligent analytics and  machine learning to help you further analyze data that's relevant to discovery requests.
 
+## Privacy
+
+The **Privacy** section on the home page shows you at a glance how your organization can build a more privacy-resilient workplace. You can generate actionable insights on enterprise personal data to help you spot issues and reduce risks. Also it Provides easy-to-use policy templates that empower employees to safeguard data, and automation features for a smoother subject rights request process.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/microsoft-365-solution-catalog.md]:
```suggestion
The **Privacy** section on the home page shows you at a glance how your organization can build a more privacy-resilient workplace. You can generate actionable insights on enterprise personal data to help you spot issues and reduce risks. Also, it provides easy-to-use policy templates that empower employees to safeguard data and automation features for a smoother subject rights request process.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by robmazz** [comment]
@yogkumgit and @msbemba  - closing, this topic needs other changes associated with the screenshot updates and the screenshots need to conform to MS style for screenshot sizing and styling. Will keep issue open and update, thanks.

**Pull request closed without merging**